lang/luajit/Portfile

```diff
@@ -31,31 +34,34 @@ patchfiles          powerpc.patch
 
 post-patch {
     reinplace "s|/usr/local|${prefix}|" ${worksrcpath}/etc/luajit.pc
-}
 
-use_configure       no
+    # Remove -march=i686 from CCOPT_x86
+    reinplace -E {s|-march=[a-z0-9_]+||g} ${worksrcpath}/src/Makefile
+
+    # The build process normally expects git and a .git directory to establish
+    # the version but it includes a fallback for a `.relver` file
+    set wrapper     [open "${worksrcpath}/.relver" w 0644]
```

Comment:
You could name the variable differently. In this case, it's not a wrapper script.

Reply:
I've changed it to `relver`